src/App.tsx

```diff
@@ -4,55 +4,69 @@ import { ThemeProvider, createTheme } from '@mui/material/styles';
 import AuthRoute from './components/auth/AuthRoute';
 import AuthState from './context/auth/AuthState';
 import Login from './pages/Login';
+import CreateProduct from './pages/inventory/CreateProduct';
 import { setAuthToken } from './utils/authUtils';
 import Home from './pages/Home';
+import Appbar from './components/common/Appbar';
+import Sidebar from './components/common/Sidebar';
+import ViewProduct from './pages/inventory/ViewProduct';
+import ViewAllProducts from './pages/inventory/ViewAllProducts';
 
 const theme = createTheme({
-    palette: {
-        primary: {
-            main: '#1F1646'
-        },
-        secondary: {
-            main: '#96694C'
-        }
+  palette: {
+    primary: {
+      main: '#1F1646'
     },
-    typography: {
-        allVariants: {
-            fontFamily: 'Poppins',
-            textTransform: 'none',
-            fontSize: 14
-        }
+    secondary: {
+      main: '#96694C'
     }
+  },
+  typography: {
+    allVariants: {
+      fontFamily: 'Poppins',
+      textTransform: 'none',
+      fontSize: 14
+    }
+  }
 });
 
 // if (localStorage.token) {
 //     setAuthToken(localStorage.token);
 // }
 
 const App = () => {
-    const token = localStorage.token;
-    React.useEffect(() => {
-        setAuthToken(localStorage.token);
-    }, [token]);
-    return (
-        <ThemeProvider theme={theme}>
-            <AuthState>
-                <Router>
-                    <Routes>
-                        <Route
-                            path='/'
-                            element={
-                                <AuthRoute redirectTo='/login'>
-                                    <Home />
-                                </AuthRoute>
-                            }
-                        />
-                        <Route path='/login' element={<Login />} />
-                    </Routes>
-                </Router>
-            </AuthState>
-        </ThemeProvider>
-    );
+  const [open, setOpen] = React.useState<boolean>(false);
```

Comment:
I initially designed the Home component to be a wrapper for the private routes so that we don't have to expose state to the main app, don't know if that might be cleaner

Reply:
To discuss further